cmake/README.md

```diff
@@ -1,7 +1,6 @@
-
 # CMake Guide
```

Comment:
Chinese docs almost finished, you can refer to: 

https://github.com/drelaptop/cocos2d-x-docs/blob/docs-update-for-317/zh/installation/CMake-Guide.md